### PR TITLE
[RF] POM>06.01_002

### PR DESCRIPTION
**Pull request by matvlesia**
https://trello.com/c/PX8bNTUV/1601-rf-pom0601002

### CODE REVIEW

File: cypress/e2e/tests/people.cy.js [hunk 1]
```diff
@@ -31,4 +32,11 @@ describe('people', () => {
             .getUserDescriptionText()
             .should('have.text', userProfileData.description);
     });
+
+    it('AT_06.01_002 | Verify People page tab is redirecting to right page',()=>{
+        homePage
+        .clickPeopleSideMenuLink()
+        .trimPeoplePageHeader()
+        .should('eq', peoplePage.headerText)
```
Comment (Rom-T) [inline cypress/e2e/tests/people.cy.js:40]: Please add 4 spaces/1 tab for methods

File: cypress/fixtures/pom_fixtures/peoplePage.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "headerText" : "People"
```
Comment (Rom-T) [inline cypress/fixtures/pom_fixtures/peoplePage.json:2]: Please rename to `peoplePageHeader` or `peoplePageTitle`

File: cypress/pageObjects/PeoplePage.js [hunk 3]
```diff
@@ -2,11 +2,17 @@ import UserProfilePage from "./UserProfilePage";
...
     getUserNameLink = () => cy.get('#people a[href*="/user/"]').contains(Cypress.env('local.admin.username'));
-
+    getPeoplePageHeader = () => cy.get('.jenkins-app-bar__content h1')
 
     clickUserNameLink() {
         this.getUserNameLink().click();
         return new UserProfilePage();
     };
+
+    trimPeoplePageHeader (){
```
Comment (Rom-T) [inline cypress/pageObjects/PeoplePage.js:12]: please rafactorinf to `trimPeoplePageHeader() {`

File: cypress/pageObjects/PeoplePage.js [hunk 4]
```diff
@@ -2,11 +2,17 @@ import UserProfilePage from "./UserProfilePage";
...
+        return this.getPeoplePageHeader().then(($el)=>{
```
Comment (Rom-T) [inline cypress/pageObjects/PeoplePage.js:13]: Please add spaces `return this.getPeoplePageHeader().then(($el) => {`

File: cypress/e2e/tests/people.cy.js (hunk 1 above)
Comment (matvlesia) [reply cypress/e2e/tests/people.cy.js]: ✅ done

File: cypress/fixtures/pom_fixtures/peoplePage.json (hunk 2 above)
Comment (matvlesia) [reply cypress/fixtures/pom_fixtures/peoplePage.json]: ✅ done

File: cypress/pageObjects/PeoplePage.js (hunk 4 above)
Comment (matvlesia) [reply cypress/pageObjects/PeoplePage.js]: ✅ done

File: cypress/pageObjects/PeoplePage.js (hunk 3 above)
Comment (matvlesia) [reply cypress/pageObjects/PeoplePage.js]: ✅ done

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by SvetlanaKanel — APPROVED** [review]

**Review by LanaBondarenko — APPROVED** [review]

**Pull request merged**